BodyWeightCalculator/src/test/java/TestCalculate.java

```diff
@@ -0,0 +1,67 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static java.awt.SystemColor.text;
+
+public class TestCalculate {
+    WebDriver driver;
+    private static final String baseUrl = "https://healthunify.com/bmicalculator/";
+
+    @BeforeMethod
+    public void SetUp() {
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/chromedriver.exe");
+        driver = new ChromeDriver();
+        driver.get(baseUrl);
+    }
+
+
+    private void InputWeightHeight(String weight, String height) {
+        driver.findElement(By.name("wg")).sendKeys(weight);
+        driver.findElement(By.name("ht")).sendKeys(height);
+        driver.findElement(By.cssSelector("[value=Calculate]")).click();
+    }
+
+    private String results() {
+        return driver.findElement(By.className("content")).getAttribute("value");
+    }
+
+    @Test
+    public void NormalWeight() {
+        InputWeightHeight("80", "191");
+        System.out.println(results());
+    }
```

Comment:
Please follow naming convention + test method should end by Test + Test without assertion it's just list of steps